docs/release_process_quarterly.md

```diff
@@ -0,0 +1,501 @@
+    - [ ] TIMs can be successfully pushed to http endpoint
+    - [ ] capable of ingesting messages via udp (see scripts in `scripts/tests` directory)
+        - [ ] TIMs
+        - [ ] BSMs
+        - [ ] SSMs
+        - [ ] SRMs
+        - [ ] SPaTs
+        - [ ] Maps
+        - [ ] PSMs
+
+
+## jpo-geojsonconverter
+### Prerequisites
+    - [ ] jpo-ode release completed
+
+### 1. Code Ready & Release Notes
+    - [ ] Release notes drafted & added to `Release_notes.md` file in `docs` directory
+    - [ ] Code changes for release are merged into `develop`
+    - [ ] A new branch `release/(year)-(quarter)` is created from `develop`
+
+### 2. Preliminary Testing
+    - [ ] code compiles
+    - [ ] unit tests pass
+    - [ ] program starts up correctly
+    - [ ] program can be configured successfully
+    - [ ] Map, SPaT, & BSM messages are consumed successfully
+    - [ ] valid ProcessedMaps, ProcessedSpats, and ProcessedMaps are outputted
+
+### 3. Project Reference Updates & Release Creation
+    - [ ] Update version number in pom.xml file for the `jpo-geojsonconverter` project if not already done.
+    - [ ] Update git submodule & artifact references for the ‘jpo-geojsonconverter’ project.
+        - [ ] Open the jpo-geojsonconverter project in an IDE.
+        - [ ] Navigate to the jpo-utils directory and run `git checkout tags/jpo-utils-x.x.x` to update the submodule reference.
+        - [ ] Update the version number in the pom.xml for the jpo-ode-core and jpo-ode-plugins dependencies to match the version number of the corresponding releases. (e.g. 1.0.0)
+        - [ ] Commit these changes to the `release/(year)-(quarter)` branch & push the changes to the remote repository.
+        - [ ] Ensure these changes pass CI/CD checks before continuing.
+    - [ ] Merge `release/(year)-(quarter)` branch into `master` branch for the jpo-geojsonconverter project, and create a release with the version number of the release. (e.g. jpo-geojsonconverter-x.x.x)
+    - [ ] Create docker image
+        - [ ] Use the release for the jpo-geojsonconverter to produce docker image with the same tag name, containing the version number.
+        - [ ] Upload docker image to [DockerHub](https://hub.docker.com/u/usdotjpoode)
+        - [ ] Tag docker image with the version number of the app. (e.g. 1.0.0)
+        - [ ] Tag docker image with year and quarter of release. (e.g. 2024-Q2)
+        - [ ] Tag docker image with 'latest' tag for the most recent release.
+    - [ ] Merge master branch into develop branch & verify that CI/CD passes.
+
+### 4. DockerHub Image Testing
+    - [ ] image starts up correctly
+    - [ ] program can be configured successfully
+    - [ ] Map, SPaT, & BSM messages are consumed successfully  
+    - [ ] valid ProcessedMaps, ProcessedSpats, and ProcessedMaps are outputted
+
+
+## jpo-conflictmonitor
+### Prerequisites
+    - [ ] jpo-ode release completed
+    - [ ] jpo-geojsonconverter release completed
```

Comment:
Nature of dependencies:

    - [ ] jpo-ode (Maven, Dockerhub) release completed
    - [ ] jpo-geojsonconverter (Maven, Dockerhub) release completed

Reply:
Addressed!